ci.not.Rune.yml

```diff
@@ -0,0 +1,57 @@
+app-id: ci.not.Rune
```

Comment:
which domain is under upstream's control rune.not.ci or not.ci or both?

Reply:
under the current appid the domain used for verification will be not.ci.

Reply:
The lead developer of this project own the `not.ci` domain.

Reply:
can I get a confirmation somehow? or are you associated with the prjoect.

Reply:
anyone can link any domain in github about, it's not verified. still needs a confirmation somehow.

Reply:
rune.not.ci also has a link to the GitHub repository in element `a[class="github-link"]`.

Reply:
Well, he also posted stuff about Rune on his blog, im.not.ci, which should be a somewhat strong indication that he fully own the not.ci domain. he is currently offline on the instant messenger I'm contact him with, if the requirement is something needs to be put on not.ci before the submission can be accepted, we probably need to wait until the next day.

Reply:
Ok that's fine we need to wait anyways. If you can add some kind of link eg the github repoof the app on not.ci then it'd be nice.

Reply:
https://not.ci/rune now redirects to the repository.

Reply:
thanks.